#goal definition
std_msgs/String folder_path
---
#result definition
bool finished
---
#feedback
int32 n_imgs
